### ISSUE TITLE
vai_q_tensorflow command gives core dumped error

### ISSUE THREAD

**Issue by YooSunYoung** [Xilinx/Vitis-AI#108]
I have run the vitis-ai-cpu latest docker image with docker_run.sh script.
I activated the vitis-ai-tensorflow conda env and then tried vai_q_tensorflow command but had this error below.
> (vitis-ai-tensorflow) :/workspace$ vai_q_tensorflow --version
> Illegal instruction (core dumped)

I also tried `import tensorflow as tf` in python console and it also had the same error.

Is there another thing to do in order to run vai_q_tensorflow?

I have tried with a prebuilt docker image and have built the image myself but ended up having same results.
The host PC is running Ubuntu 16.04. 
I was following the instruction on the readme and I wanted to try  `CIFAR10 Classification using Vitis AI and TensorFlow` tutorial.

**Comment by xieyi4650** [comment]
@YooSunYoung Hi, I have tried both build with docker_run.sh script to build docker image, and pull image from docker hub. And all run correctly.  So I can not reproduce your result.
Generally speaking,  if you get docker image correctly there is nothing more to do.  Maybe you can check you docker following [Vitis-AI-installation](https://github.com/Xilinx/Vitis-AI#installation)

**Comment by mkettn** [comment]
Hi,
I have the same problem with Ubuntu 18.04. Both with the docker image from the hub and with a self built one. 

It looks like the tensorflow library installed by default does not work, but if I execute `pip install --force-reinstall tensorflow==1.15.2` inside the docker image command line ... at least training works.

**Comment by YooSunYoung** [comment]
@xieyi4650 Hi! Sorry for the poor information I put on this issue. However, I guess the problem was due to a lack of memory. Cause I tried again with a different computer which has better RAM and it worked without any problem. Thank you for answering my question.

@mkettn Yes, I tried that too and I still couldn't use `vai_q_tensorflow` command. But I don't have any problem with a new computer now. Thank you for your help! :D
